templates/directus.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  AppService,
+  createTemplate,
+  PostgresService,
+  MySQLService,
+  RedisService,
+  randomPassword,
+  randomString,
+} from "~templates-utils";
+
+export default createTemplate({
+  name: "Directus",
+  schema: {
+    type: "object",
+    required: ["projectName", "domain", "appServiceName", "adminEmail", "adminPassword", "databaseType", "databaseServiceName", "redisServiceName"],
+    properties: {
+      projectName: {
+        type: "string",
+        title: "Project Name",
+      },
+      domain: {
+        type: "string",
+        title: "Domain",
+      },
+      appServiceName: {
+        type: "string",
+        title: "App Service Name",
+        default: "directus",
+      },
+      adminEmail: {
+        type: "string",
+        title: "Admin E-Mail",
+        description: "admin@example.com",
+      },
+      adminPassword: {
+        type: "string",
+        title: "Admin Password",
+      },
+      databaseType: {
+        type: "string",
+        title: "Database Type",
+        oneOf: [
+          { enum: ["postgres"], title: "Postgres" },
+          { enum: ["mysql"], title: "MySQL" },
+        ],
+      },
+      databaseServiceName: {
+        type: "string",
+        title: "Database Service Name",
+        default: "db",
+      },
+      redisServiceName: {
+        type: "string",
+        title: "Redis Service Name",
+        default: "redis",
+      },
+    },
+  } as const,
+  generate({
+    projectName,
+    domain,
+    appServiceName,
+    adminEmail,
+    adminPassword = randomPassword(),
+    databaseType,
+    databaseServiceName,
+    appKey = randomString(48),
+    appSecret = randomString(48),
+    redisServiceName,
+  }) {
+    let databasePassword = randomPassword();
```

Comment:
use const

Reply:
Fixed in [27974ce](https://github.com/easypanel-io/templates/pull/12/commits/27974cef460b51c12f16f7d0fbe0ac4a2912a67c)